projects/kit/src/lib/masks/number/number-mask.ts

```diff
@@ -53,13 +54,19 @@ export function maskitoNumberOptionsGenerator({
     thousandSeparator?: string;
     prefix?: string;
     postfix?: string;
+    minusSign?: string;
 } = {}): Required<MaskitoOptions> {
-    const pseudoMinuses = [
+    const pseudoMinuses: string[] = [
         CHAR_HYPHEN,
         CHAR_EN_DASH,
         CHAR_EM_DASH,
         CHAR_JP_HYPHEN,
-    ].filter(char => char !== thousandSeparator && char !== decimalSeparator);
+        CHAR_MINUS,
+        minusSign,
+    ].filter(
+        char =>
+            char !== thousandSeparator && char !== decimalSeparator && char !== minusSign,
```

Comment:
Probably, we should should not put `minusSign` inside `pseudoMinuses` if we will anyway filter it later 😄